### PR TITLE
F [#PS-136] add captured attribute to Charge

### PR DESCRIPTION
**Pull request by sadtuna**
https://shore.myjetbrains.com/youtrack/issue/PS-136

### CODE REVIEW

**Comment by sadtuna** [comment]
ping @ceigel

File: lib/payment/merchant_connector.rb [hunk 1]
```diff
@@ -162,7 +162,8 @@ def base_path
 
     # :nodoc:
     module CreateChargeParams
-      REQUIRED_PARAMS = [:credit_card_token, :amount_cents, :currency].freeze
+      REQUIRED_PARAMS = [:credit_card_token, :amount_cents, :currency,
+                         :captured].freeze
```
Comment (ceigel) [inline lib/payment/merchant_connector.rb]: stripe parameter name is 'capture' not 'captured'

Comment (sadtuna) [inline lib/payment/merchant_connector.rb]: Yes, you're right, they use 'captured' only when thecharge is already created. I'll fix using 'capture' attribute for creating a Charge, but will use 'captured' when it's just a read-only attribute (and will use 'captured' in the db too). What do you think?

Comment (ceigel) [inline lib/payment/merchant_connector.rb]: Hmmm, sounds complicated ... But ok, let's try it

**Comment by ceigel** [comment]
👍

**Pull request merged**